contents/docs/libraries/js/index.mdx

```diff
@@ -522,9 +522,10 @@ In this section we describe some additional details on advanced configuration av
 | `advanced_disable_decide`<br/><br/>**Type:** Boolean<br/>**Default:** `false` | Will completely disable the `/decide` endpoint request (and features that rely on it). More details below.     |
 | `advanced_disable_feature_flags`<br/><br/>**Type:** Boolean<br/>**Default:** `false` | Will keep `/decide` running, but without any feature flag requests     |
 | `advanced_disable_feature_flags_on_first_load`<br/><br/>**Type:** Boolean<br/>**Default:** `false` | Stops from firing feature flag requests on first page load. Only requests feature flags when user identity or properties are updated, or you manually request for flags to be loaded.  |
+| `custom_campaign_params` <br/><br/>**Type:** Array<br/>**Default:** `[]`      | List of query params to be automatically captured (see [UTM Segmentation](/docs/data/utm-segmentation) ) |
 | `feature_flag_request_timeout_ms`<br/><br/>**Type:** Integer<br/>**Default:** `3000` | Sets timeout for fetching feature flags  |
+| `person_processing`<br /><br />**Type:** Enum: `always`, `identified_only`, `never` | Set whether events should have person processing enabled (see [Events](/docs/data/events#events-with-person-processing)) |
```

Comment:
can we add this to the node, RN, PHP sdk docs too please? (wherever it's applicable)

- https://posthog-ddjv5eybg-post-hog.vercel.app/docs/libraries/node#options 
- https://posthog-ddjv5eybg-post-hog.vercel.app/docs/libraries/react-native#configuration-options 
- https://posthog-ddjv5eybg-post-hog.vercel.app/docs/libraries/php#config-options